pyproject.toml

```diff
@@ -61,3 +61,23 @@ tensorflow-io-gcs-filesystem = [
     { version = "< 0.32.0", markers = "platform_system == 'Windows'" }
 ]
 resampy = "^0.4.2"
+
+
+[tool.semantic_release]
+version_variable = "pyproject.toml:version"
+tag_format = "v{version}"
+commit_parser = "angular"
+commit_message = "test new message"
```

Comment:
This seems wrong - might I suggest `"{version}\n\nAutomatically generated by python-semantic-release"`?